maintainers/maintainer-list.nix

```diff
@@ -17935,6 +17935,11 @@
     githubId = 73759599;
     name = "Yaya";
   };
+  yboettcher = {
+    name = "Yannik Böttcher";
```

Comment:
it would be cool to add another mean of contact

Reply:
would be nice if you have a matrix to add that but optional